src/clue.js

```diff
@@ -1,13 +1,143 @@
 // ITERATION 1
+function selectRandom(arr) {
+  return arr[Math.floor(Math.random() * Math.floor(arr.length))];
+}
+function pickMystery() {
+  const objectWithCards = {
+    suspect: selectRandom(suspectsArray),
+    weapon: selectRandom(weaponsArray),
+    room: selectRandom(roomsArray)
+  };
+  return objectWithCards;
+}
```

Comment:
You don't have to create an object like this actually. You can rather just write:
`const newObject = {}`
and then assign the new proprietors to it.